services/src/unit-test/java/uk/gov/moj/sdt/services/BulkFeedbackServiceTest.java

```diff
@@ -149,6 +118,14 @@ public void setUp() {
 
     @Test
     void testGetBulkFeedback() {
+
+        final IGlobalParameter globalParameterData = new GlobalParameter();
+        globalParameterData.setName(IGlobalParameter.ParameterKey.DATA_RETENTION_PERIOD.name());
+        globalParameterData.setValue("90");
+        when(mockGlobalParameterCache.getValue(IGlobalParameter.class,
+                                               IGlobalParameter.ParameterKey.DATA_RETENTION_PERIOD.name())).thenReturn(globalParameterData);
+        bulkFeedbackService = new BulkFeedbackService(mockBulkSubmissionDao, mockGlobalParameterCache);
```

Comment:
Move the intialisation of the bulkFeedbackService field back into the setUp() method so it can be used by all tests.

Reply:
done